src/main/java/com/zufar/onlinestore/user/converter/AddressDtoConverter.java

```diff
@@ -0,0 +1,25 @@
+package com.zufar.onlinestore.user.converter;
+
+import com.zufar.onlinestore.user.dto.AddressDto;
+import com.zufar.onlinestore.user.entity.Address;
+import org.springframework.stereotype.Service;
+
+@Service
```

Comment:
Use `@Component` here. Also I suppose that we will create a separate factory for more efficient bean management in the future